### ISSUE TITLE
Welcome!

### ISSUE THREAD

**Issue by mikeal** [iojs/iojs-nl#1]
Welcome, new localizations team :)

To get this team up and running you have a few initial tasks to accomplish:
- Create a README.md for this repository, it should include:
  - A "call to action" pointing to the Issue tracker for new translators to find tasks.
  - A list of the team members.
  - Instructions on how to log an issue to be added to this team.
- Set the repository description. This should not be in English, it should be in the language for this community :)
- Social Media
  - Create a twitter account for this language. Share the password privately with the team.
  - Create accounts on other social media popular in this language. Share the passwords private with the rest of the team.
  - Make sure you use these accounts not just to promote newly translated material but also to attract people in your community to help with translation tasks.
- _Optional:_ If you plan to do a Google hangout w/ your localization team have @mikeal add you to the Google+ account so that it can post to the `iojs` YouTube channel. Once you are added there is [documentation on how to setup a hangout-on-air](https://github.com/iojs/io.js/wiki/Meetings:-Creating-a-Hangouts-On-Air-Event).
- Finally, create a new Issue to translate and publish the [latest io.js post](https://medium.com/@iojs). Once translated you should post it wherever you think it will get the most visibility to the community speaking you language, you don't have to post it to Medium unless you think that it is best.

**Comment by timdp** [comment]
Looks like @yoshuawuyts (hi, by the way) beat me to it. :)

I just created a Twitter account called @iojsnl. I'll decorate it further after this comment and email the password to Yoshua.

So an updated todo list:
- Add a CTA that points to the issue tracker.
- Translate https://medium.com/node-js-javascript/io-js-week-of-february-6th-2015-e185388549a4 to Dutch.
- Optional: set up Hangouts.

**Comment by timdp** [comment]
I see the Norwegian account is called @iojs_no. Should I add an underscore to ours as well then?

**Comment by mikeal** [comment]
@timdp entirely up to you :)

**Comment by mikeal** [comment]
but, let me know if you are because I want to tweet out from the official_iojs to follow you :)

**Comment by yoshuawuyts** [comment]
hey @timdp! :D I actually was going for a PR, but apparently you can't do that if the repo is empty... so I figured I'd just push haha. Anyway, I think your list is looking good. I'll try to wrap up the readme with the points outlined above. I'll PR them this time though, hah.

re underscore: I think it's common practice to add an underscore?

**Comment by yoshuawuyts** [comment]
Oh, and nice work on setting up twitter so fast ^^. Guess you beat me to that hah.

**Comment by timdp** [comment]
I'm not a huge fan of underscores, but I've nevertheless added one for consistency. Password coming your way now!

**Comment by AdriVanHoudt** [comment]
I was just added and wanted to say hi! I'm from Belgium and a native Dutch speaker. So if I can help just ask!

**Comment by yoshuawuyts** [comment]
@AdriVanHoudt welcome! If you want you can PR yourself into the `README.md`. We're aiming to release translations of the newsletter every weekend. Apart from that we don't have any concrete plans for what can / should be translated, but feel free to take initiative!

**Comment by AdriVanHoudt** [comment]
@yoshuawuyts ok cool thanks!

**Comment by AdriVanHoudt** [comment]
done see #4

**Comment by yoshuawuyts** [comment]
@AdriVanHoudt ^^

**Comment by timdp** [comment]
Hi, Adri!

**Comment by AdriVanHoudt** [comment]
Hi, Tim!

**Comment by AdriVanHoudt** [comment]
Close this or leave it open for future welcomes?

**Comment by AdriVanHoudt** [comment]
I don't know if this is worth a new issue so I'll post this here https://github.com/iojs/io.js/issues/978
Personally I'm in full favor of 1 'version' of _Node_

**Comment by timdp** [comment]
Ping.

I was pretty busy as well as sick for about two weeks, but I should now be able to free up some time again. Looks like we're accumulating a bit of a backlog. What do you guys think I/we should get started on first?

**Comment by yoshuawuyts** [comment]
@timdp yeah, also been super busy. Frankly coding is more fun than translating, haha. If you want to give anything a spin, the website might be a cool one to do. Cheers!

**Comment by AdriVanHoudt** [comment]
They are planning to add a "capture" page for translations so when a person wants to see a certain page translated but it is not there yet we'll get notified but I don't know when that is actually coming. As for general translations the website seems like the primary thing to translate.
